Jenkinsfile

```diff
@@ -685,7 +686,16 @@ pipeline {
                             steps {
                                 powershell '''
                                 $ErrorActionPreference = 'Stop'
-                                .\\hack\\ci\\windows.ps1
+                                [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+                                Invoke-WebRequest https://github.com/jhowardmsft/docker-ci-zap/blob/master/docker-ci-zap.exe?raw=true -OutFile C:/Windows/System32/docker-ci-zap.exe
+                                $env:SOURCES_DRIVE="d"
+                                $env:SKIP_VALIDATION_TESTS="true"
+                                $env:SOURCES_SUBDIR="gopath"
+                                $env:TESTRUN_DRIVE="d"
+                                $env:TESTRUN_SUBDIR="CI-$env:BUILD_NUMBER"
+                                $env:WINDOWS_BASE_IMAGE="mcr.microsoft.com/windows/servercore"
+                                $env:WINDOWS_BASE_IMAGE_TAG="ltsc2016"
```

Comment:
Is there an advantage to setting these environment variables in powershell instead of using something like;

```groovy
environment {
    DOCKER_BUILDKIT        = '0'
    SKIP_VALIDATION_TESTS  = '1'
    SOURCES_DRIVE          = 'd'
    SOURCES_SUBDIR         = 'gopath'
    TESTRUN_DRIVE          = 'd'
    TESTRUN_SUBDIR         = $(BUILD_NUMBER)
    WINDOWS_BASE_IMAGE     = 'mcr.microsoft.com/windows/servercore'
    WINDOWS_BASE_IMAGE_TAG = 'ltsc2016'
}
```

Reply:
All right, that looks better. I'll update the Jenkinsfile